miniserv.pl

```diff
@@ -2101,12 +2106,8 @@ sub handle_request
 			$method = "GET";
 			$querystring .= "&failed=".&urlize($failed_user)
 				if ($failed_user);
-			if ($twofactor_msg) {
```

Comment:
Is there a reason we don't need these kinds anymore?

Reply:
Yeah, we don’t need to send the password back or keep track of failed attempts anymore, since the form is now submitted over an HTTP request without reloading the page and losing the password and other data.